cudax/include/cuda/experimental/__stf/places/exec/host/callback_queues.cuh

```diff
@@ -320,8 +320,7 @@ inline void cudagraph_callback_dispatcher(void* userData)
 }
 
 // There is likely a more efficient way in the current implementation of callbacks !
-template <int = 0> // template to make this `inline` without using `inline`, which nvcc dislikes
-__global__ void callback_completion_kernel(int* completion_flag)
+__global__ static void callback_completion_kernel(int* completion_flag)
```

Comment:
I believe that's what [this macro](https://github.com/NVIDIA/cccl/blob/main/libcudacxx/include/cuda/std/__cccl/visibility.h#L87) is for:

```suggestion
CCCL_DETAIL_KERNEL_ATTRIBUTES void callback_completion_kernel(int* completion_flag)
```

Reply:
We cannot use visibility macros ebcause nvcc complains there

Reply:
Really? So what's the purpose of that macro? I thought we used this to get around similar issues in CUB?